server/index.js

```diff
@@ -22,6 +24,42 @@ app.get('/products', (req, res, next) => {
   });
 });
 
+app.get('/product/:productId', (req, res, next) => {
+  if (!req.params.productId) {
+    throw new ClientError(400, 'productId is required');
+  }
+  const options = {
+    method: 'GET',
+    url: `https://dummyjson.com/products/${req.params.productId}`
+  };
+  axios.request(options).then(response => {
```

Comment:
You're making a network request to get the products? I thought you'd read them out of the database. Or maybe that's a later feature?